makeservices/makemysql

```diff
@@ -8,4 +8,9 @@ PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real | tee /dev/tty
 
 echo 'Changing WordPress database password'
 cd ~/public_html/
-wp config set DB_PASSWORD "$PASS" > /dev/null
+FILE=wp-includes/version.php
+if [ -f "$FILE" ]; then
+    wp config set DB_PASSWORD "$PASS" > /dev/null
+else
+    echo "WordPress is not installed!"
```

Comment:
```suggestion
    echo "WordPress is not installed."
```
Nit: IMO having an exclamation mark here is somewhat misleading, as not having WordPress installed is not a strange or unexpected condition.

Reply:
Check the PR down below, luckily you didn't make a PR also lool